python-sdk/actionml/__init__.py

```diff
@@ -398,9 +400,9 @@ def send_query(self, data):
 
 
 class CommandClient(BaseClient):
```

Comment:
should be `CommandsClient` plural, we should change all java and python "client" classes to follow this naming since they deal with resource types, which are collections.